### ISSUE TITLE
COTT Doesn't run

### ISSUE THREAD

**Issue by Alpeel** [HearthProject/OneClient#40]
[Image here](http://prntscr.com/gdwanz) I tested a few other modpacks and they seem to be working, I just get the error with Craft of the Titans 1.2.

**Comment by Nodachi216** [comment]
I couldn't reproduce this. Craft of the Titans downloaded, installed, and ran fine. The error log you posted shows that one mod, Agricraft, caused both crashes, so it could have been a glitch on Curse's end. I'd recommend deleting the instances and trying again.

**Comment by modmuss50** [comment]
Yeah, I agree. The crash you posted seems to be an issue with the mod, it would be very unlikely for the launcher to cause that crash.

**Comment by Alpeel** [comment]
Opening the same modpack with the same version in Twitch launcher works perfectly though.

**Comment by Nodachi216** [comment]
It was probably a download glitch then. Have you tried deleting the instances and letting 1C try again?

**Comment by Alpeel** [comment]
Tried it with the modpacks I mentioned, still the same problem.
